src/java/org/jivesoftware/openfire/session/Session.java

```diff
@@ -212,4 +213,12 @@
      * @return cipher suite name.
      */
     public String getCipherSuiteName();
+
+    /**
+     * Returns the language code that should be used for this connection

```

Comment:
Might prefer "session" here rather than "connection" (also below), but not a blocker.